Dockerfile

```diff
@@ -0,0 +1,31 @@
+FROM ubuntu:latest
+
+ENV TZ=Europe/Kiev
```

Comment:
I don't live in Kiev, but after building an image with this Dockerfile, the time inside of my container seems to be the time in Kiev:
![image](https://user-images.githubusercontent.com/23319866/98221624-ed179800-1f60-11eb-9133-5ce8783a1852.png)